packages/workflow/src/errors/node-operation.error.ts

```diff
@@ -15,6 +17,9 @@ export class NodeOperationError extends NodeError {
 	) {
 		if (typeof error === 'string') {
 			error = new Error(error);
+		} else if (!(error instanceof ApplicationError)) {
+			// this error was no processed by n8n, obfuscate error message
+			error.message = OBFUSCATED_ERROR_MESSAGE;
```

Comment:
We should stop overwriting error properties like this, as that remove context on errors that get reported to Sentry.